sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -470,6 +473,16 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    public partial class PipelineContext
+    {
+        internal PipelineContext() { }
+        public System.DateTimeOffset CreatedOn { get { throw null; } }
+        public int RetryAttempt { get { throw null; } set { } }
+    }
+    public enum PipelinePolicyReplacement
+    {
+        RetryPolicy = 0,
```

Comment:
We can expand this list if needed.

Reply:
I think this is an interesting pattern for how to grow the list of replaceable policies. Pragmatically, I'm not sure we would grow it beyond RetryPolicy though.

Reply:
Why don't we use HttpPipelinePosition and just add RetryPolicy value

Reply:
That is an interesting idea. We could probably even not add the new SetPolicy method as a user could just call AddPolicy and pass in RetryPolicy value. It does muddy the semantics somewhat though.

Reply:
Yeah, I would worry about anyone calling AddPolicy now being able to override the RetryPolicy easily if we added it to HttpPipelinePosition.  I liked that AddPolicy was more causal and Replace/Set policy was more advanced.

Reply:
I mean, they are sort of adding the policy so maybe it is okay. They are adding a retry policy and the constraints are that there is only 1 retry policy.